tests/Util/PHP/AbstractPhpProcessTest.php

```diff
@@ -59,7 +59,7 @@ public function testShouldUseGivenSettingsToCreateCommand()
             'display_errors=1',
         ];
 
-        $expectedCommandFormat  = "%s -d %callow_url_fopen=1%c -d %cauto_append_file=%c -d %cdisplay_errors=1%c";
+        $expectedCommandFormat  = '%s -d %callow_url_fopen=1%c -d %cauto_append_file=%c -d %cdisplay_errors=1%c';
```

Comment:
This change is unrelated to enabling the `ordered_imports` fixer, but follows [`.php_cs#L67`](https://github.com/sebastianbergmann/phpunit/blob/92cba1d0bce5fd91ef74a49c31a8cbf1a42a79ef/.php_cs#L67).